src/components/QuranReader/TranslationView/TranslationViewCell.module.scss

```diff
@@ -1,7 +1,26 @@
 @use "src/styles/breakpoints";
 
 .highlightedContainer {
-  background: var(--color-background-alternative-faded);
+  --color-gradient-transition: 196, 196, 196;
+
+  [data-theme="dark"] & {
+    --color-gradient-background-dark: 150, 125, 225;
+    background: radial-gradient(ellipse farthest-corner at -360% -360%, rgba(var(--color-gradient-background-dark), 0.25) 50%, rgba(var(--color-gradient-transition), 0) 90%);
```

Comment:
We already have pre-defined tokens for background colors for each theme:

<img width="863" alt="Screen Shot 2021-12-20 at 18 58 24" src="https://user-images.githubusercontent.com/15169499/146763927-ea8baec4-9117-4294-adbe-caafd7a8cf96.png">